README.md

```diff
@@ -53,18 +53,22 @@ import (
   "github.com/aws/aws-xray-sdk-go/xray"
 
   // Importing the plugins enables collection of AWS resource information at runtime.
-  // Every plugin should be imported after "github.com/aws/aws-xray-sdk-go/xray" library.
-  _ "github.com/aws/aws-xray-sdk-go/plugins/ec2"
-  _ "github.com/aws/aws-xray-sdk-go/plugins/beanstalk"
-  _ "github.com/aws/aws-xray-sdk-go/plugins/ecs"
+  "github.com/aws/aws-xray-sdk-go/plugins/ec2"
```

Comment:
Could you please rename these three plugin package since directly changing the plugin behavior under the same package name will not result in any compile or runtime errors for consumers, and will silently fail to record AWS metadata on their segments. Like `"github.com/aws/aws-xray-sdk-go/awsplugins/ec2"`